tools/pkglint/lib/util.ts

```diff
@@ -22,7 +22,7 @@ export function expectJSON(pkg: PackageJson, jsonPath: string, expected: any, ig
     }
 
     function applyCaseInsensitive(val: any): any {
-        if (!caseInsensitive || val == null) { return val; }
+        if (!caseInsensitive || val == null) { return JSON.stringify(val); }
```

Comment:
Why? And if there's a good reason, you could change the return type to `string`.

Reply:
Okay. changed return type to `string` (also in `applyIgnore`). The reason is that users would deep-equal (`===`) these values and that won't work if they are not primitives.